README.md

```diff
@@ -1,5 +1,5 @@
 # Obsidian Slab
 
-Literally just adds the Obsidian slab from Consistency+ as a seprate block.
+Literally just adds the Obsidian slab from Consistency+ as a separate block.
 
-IMS was the guy in charge of alot of the actual code when making the inital block, Siuol took care of the separation after the fact and maintains this specific mod.
+IMS was the guy in charge of a lot of the actual code when making the initial block, Siuol took care of the separation after the fact and maintains this specific mod.

```

Comment:
If you're going to fix up some grammar here, don't do a half-arsed job at it; remove the comma splice too :P

Reply:
I think I fixed it, assuming I understood correctly what you meant.